components/exercises/chapter-4-oscar/Utils.tsx

```diff
@@ -70,3 +72,139 @@ export function reverseArrayInPlace(arr: any[]): undefined {
     // After the first iteration, the array becomes [5, 2, 3, 4, 1]
   }
 }
+
+/**
+ * Represents the properties of a list item.
+ */
+export type ListItemProp = {
+  value: ReactNode;
```

Comment:
Shouldn't this be a string?

Reply:
The parameter type I was passing was ReactNode[], so I had to match it with the value: ReactNode. I have updated the parameter type to string[] | number [], and now it is working